functions/New-DbaDbTable.ps1

```diff
@@ -455,16 +455,28 @@ function New-DbaDbTable {
                         $object.Columns.Add($sqlColumn)
                     }
 
+                    # user has specified a schema that does not exist yet
+                    if (-not ($db | Get-DbaDbSchema -Schema $schema -IncludeSystemSchemas)) {
+                        $schemaObject = New-Object -TypeName Microsoft.SqlServer.Management.Smo.Schema $db, $schema
```

Comment:
Can we use the New-DbaDbSchema function you created here?

Reply:
Glad you asked, I was originally hoping to do that, but then noticed the -Passthru logic implies that a caller could be scripting out the definitions and not actually creating the objects. So, I decided to use the SMO Schema object which could then be used in either the -Passthru scenario or the default scenario when the objects are actually created. 

I'm open to changing it so let me know your thoughts on this.

Reply:
This looks great, thank you. If you decide with using the other command, you can submit a PR for that.